docs/advanced.md

```diff
@@ -290,11 +290,12 @@ can either move your module to a different directory or use the `exclude` or
 
 - `include`: RegExp or function
 
-    Allow list certain modules. See [`require-directory`](https://www.npmjs.com/package/require-directory) for details.
+
+    Allow list certain modules. Either a regex or callback can be provided. `true` = load the file.
 
 - `exclude`: RegExp or function
 
-    Block list certain modules. See [`require-directory`](https://www.npmjs.com/package/require-directory) for details.
+    Block list certain modules. Either a regex or callback can be provided. `false` = do not load the file.
```

Comment:
I am not sure what ``false` = do not load the file.` means. If it is the callback, it looks inverted. (I have not checked the code or runtime, just reading the docs alone at the moment.)

Reply:
You're correct, it should be return `true`. Fixed.